lib/plist/generator.rb

```diff
@@ -143,13 +141,13 @@ def self.tag(type, contents = '', &block)
   def self.wrap(contents)
     output = ''
 
-    output << '<?xml version="1.0" encoding="UTF-8"?>' + "\n"
-    output << '<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">' + "\n"
-    output << '<plist version="1.0">' + "\n"
+    output << '<?xml version="1.0" encoding="UTF-8"?>' << "\n"
+    output << '<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">' << "\n"
+    output << '<plist version="1.0">' << "\n"
```

Comment:
I think the `+` and `<<` syntaxes are both a bit confusing. What do you think about using a heredoc string instead?

```ruby
output <<-HEADER
<?xml version="1.0" encoding="UTF-8"?>
...etc
HEADER
```